Composer/packages/client/src/router.tsx

```diff
@@ -48,7 +48,8 @@ const Routes = props => {
                 <Router basepath={BASEPATH} {...parentProps}>
                   <Redirect from="/" to={resolveToBasePath(BASEPATH, 'dialogs/Main')} noThrow />
                   <SettingPage path="setting/*" />
-                  <LUPage path="language-understanding/*" />
+                  <Redirect from="language-understanding" to="language-understanding/all" noThrow />
```

Comment:
do we need to check if a lu file exist? Currently, if the url is language-understanding/hhh, it still shows the language-understanding page. probably 404 or redirect to lanugage-understanding/all?

Reply:
Nice catch, sure will update it